images/entrypoint.sh

```diff
@@ -52,7 +52,7 @@ if [ "$2" == "origin" ]; then
 
         shopt -s nullglob
         for fullfile in /opt/scitokens-server/etc/trusted-cas/*.pem; do
-            echo "Importing CA certificate $fullfile into the Java trusted CA store."
+            #echo "Importing CA certificate $fullfile into the Java trusted CA store."
```

Comment:
Same with this and with lines 89, 92 & 96-- rather than comment it out, just delete the line.